server.js

```diff
@@ -1,29 +1,5 @@
 const logger = require('./log')
-const config = require('./server/config')
 const app = require('./server/index')
-const healthcheck = require('./server/services/healthcheck')
-const appInsights = require('./azure-appinsights')
-const { flattenMeta } = require('./server/misc')
-
-if (config.healthcheckInterval) {
-  reportHealthcheck()
-  setInterval(reportHealthcheck, config.healthcheckInterval * 60 * 1000)
-}
-
-function reportHealthcheck() {
-  healthcheck(recordHealthResult)
-}
-
-function recordHealthResult(err, results) {
-  if (err) {
-    logger.error('healthcheck failed', err)
-    return
-  }
-  logger.info('healthcheck', results)
-  if (results.healthy && appInsights) {
-    appInsights.defaultClient.trackEvent('healthy', flattenMeta(results))
-  }
```

Comment:
when enabled, this polls and write health status events to app insights but this isn't enabled anywhere.
Removing this as if we required health status tracking we could implement a solution globally rather than embed in each app